packages/ckeditor5-ui/src/panel/balloon/contextualballoon.ts

```diff
@@ -320,10 +323,35 @@ export default class ContextualBalloon extends Plugin {
 			this._visibleStack.get( this.visibleView! )!.position = position;
 		}
 
-		this.view.pin( this._getBalloonPosition()! );
+		this.view.pin( this.getPosition()! );
 		this._fakePanelsView!.updatePosition();
 	}
 
+	/**
+	 * Returns position options of the last view in the stack.
+	 * This keeps the balloon in the same position when the view is changed.
+	 */
+	public getPosition(): Partial<PositionOptions> | undefined {
```

Comment:
Maybe we could rename this method as it is now public to something that would not confuse users. Maybe `getPositionOptions()` would better reflect what it does.